Automattic/jetpack
# JITMs: remove pre-connection JITMs and only load scripts on Jetpack and Woo pages

## Proposed changes:

This PR does 2 things:

- It introduces a new method to detect Jetpack and Woo admin pages, so the JITM scripts are only enqueued on those pages. We do not want to display JITMs anywhere else.
- It removes JITMs that were hardcoded in the Jetpack plugin and still displayed on other pages in wp-admin.

### Other information:


## Jetpack product discussion

* p1HpG7-r5K-p2

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

1. Start with a brand new site, not connected to WordPress.com yet.
2. Go to Appearance > Themes, install and activate a classic theme like Twenty Ten.
3. Go to Plugins > Add New, and add the Classic Widgets plugin.
4. Go to Appearance > Widgets
    * You should see a JITM there
5. Go to the main dashboard, `/wp-admin/`
6. Open the JavaScript console.
    * `jitm_config` should be an object with info that can be used by JITMs.
 7. Now switch to the branch from this PR.
 8. Repeat steps 4 to 6
     * The JITM and the object should be gone.
